quality-metrics/qm-007-integration-tests.md

```diff
@@ -22,5 +22,5 @@ Integration testing is pinnacle to velocity and verifying community cookbooks do
 Pseudocode or actual code that can be used to automatically verify the rule and/or assign appropriate points.
 
 ```bash
-   [ $(find test/integration/default/ -name '*_spec.rb' | wc -l) -ge $(find recipes/ -name '*.rb' | wc -l ) ]
+   [ $(find test/integration/ -name '*_spec.rb' | wc -l) -ge $(find recipes/ -name '*.rb' | wc -l ) ]
```

Comment:
Possibly.

Implementation here gets a little tricky because tests aren't assumed to be included in the cookbook tarball. So we'll either have to clone the source URL to use the super convenient `find` and `wc` utilities or use the GitHub API to walk [a JSON representation of the test and recipe directory trees](https://developer.github.com/v3/git/trees/#get-a-tree-recursively).